corehq/apps/users/__init__.py

```diff
@@ -1,4 +1,17 @@
 from corehq.preindex import ExtraPreindexPlugin
+from django.apps import AppConfig
 from django.conf import settings
 
-ExtraPreindexPlugin.register('users', __file__, settings.NEW_USERS_GROUPS_DB)
+from .signals import connect_user_signals
+
+
+class UsersAppConfig(AppConfig):
+    name = 'corehq.apps.users'
+
+    def ready(self):
+        """Code to run with Django starts"""
+        ExtraPreindexPlugin.register('users', __file__, settings.NEW_USERS_GROUPS_DB)
+        connect_user_signals()
+
+
+default_app_config = 'corehq.apps.users.UsersAppConfig'
```

Comment:
According to the docs, you can just change the `INSTALLED_APPS` entry to point to this class, but I kept getting a weird, cryptic error when I tried that - I think we do other things with `INSTALLED_APPS`, and changing this broke something non-obvious.  This seems to work, though.